### PR TITLE
Introduce EncryptedWalletHandler

### PR DESCRIPTION
**Pull request by Sajma27**
# Summary
A description of the changes proposed in the pull request.

Fixes #number

## Checklist
- [ ] Change is small and easy to review (please split big changes into multiple PRs)
- [ ] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [ ] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-relayer/test/core/services/EncryptedWalletHandler.test.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import {expect} from 'chai';
+
+describe('INT: EncryptedWalletHandler', () => {
+  const notConfirmedEmail = 'example@email.com';
+  const confirmedEmail = 'confirmed@email.com';
+  let encryptedWalletHandler: EncryptedWalletHandler;
+  let encryptedWallet: EncryptedWallet;
+
+  before(() => {
+    const knex = getKnexConfig();
```
Comment (xitronix) [inline universal-login-relayer/test/core/services/EncryptedWalletHandler.test.ts:15]: destroy knex instance at the end of test

Comment (Sajma27) [reply universal-login-relayer/test/core/services/EncryptedWalletHandler.test.ts]: Done

**Comment by xitronix** [comment]
I think you should introduce endpoint for this service too

**Comment by Sajma27** [comment]
> I think you should introduce endpoint for this service too

**Pull request merged**